docs/BuildingFromSource.html

```diff
@@ -170,39 +170,8 @@ <h3>Building smaller binaries</h3>
 
 <h3 id=Portability>Build portability</h3>
```

Comment:
Perhaps remove this entire section? This is what people expect anyway. The section was necessary because the previous behavior was different from this common expectation.